modules/openxoutstreamBidAdapter.md

```diff
@@ -18,16 +18,14 @@ This bid adapter supports Banner.
 var adUnits = [
   {
     code: 'test-div',
-    sizes: [[300, 250]],  // a display size
+    sizes: [[300, 250],[640,480]],  // a display size
```

Comment:
First, please update this documentation to use the `mediaTypes` syntax -- `sizes` here is old. (Yes, there are many adapters that haven't updated this yet, but we have to start somewhere :-)

But more importantly, this specific change isn't making sense to me: why are you passing in a common display size and a video player size at the same time?

Reply:
no problem, sorry I missed that the syntax had updated!

The 2nd size was in there by mistake. I was testing something and forgot to delete it.

Thanks!